wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -291,7 +292,7 @@ def querystring(context, **kwargs):
 
 
 @register.simple_tag(takes_context=True)
-def table_header_label(context, label=None, sortable=True, ordering=None, sort_context_var='ordering', sort_param='ordering', sort_field=None):
+def table_header_label(context, label=None, sortable=True, ordering=None, sort_context_var='ordering', sort_param='ordering', sort_field=None, parent_page_title=None):
```

Comment:
This tag is intended to be reusable across all listings in the admin, not just page listings, so there should not be any page-specific code here. I suggest making this into an optional `title` parameter - the code that calls this tag is then responsible for building up the title string.

Reply:
@gasman currently this tag is only called from `wagtail/admin/templates/wagtailadmin/pages/listing/_table_headers_explore.html`, which isn't in a shared folder. If I were to move the logic of building up the string to the template it would mean a lot of logic in the template, for example the logic currently in the `if ordering == sort_field` would need to be duplicated in the template - I'm not sure it is even possible.

Reply:
Have now pushed fbc3f922bc240506a27aa297cb96e3f2d6402d85 to address this. Yep, building up the string within the template isn't really doable, so I've added a new tag `page_table_header_label` to wrap the `table_header_label` tag and keep the latter generic.

Reply:
Thanks @gasman